keyboards/themadnoodle/ncc1701kb/v2/readme.md

```diff
@@ -1,17 +1,27 @@
-# NCC-1701-KB v2 Keypad with Encoder
+# NCC-1701-KB v.2
 
-![NCC-1701-KB V2 TOP](https://static.wixstatic.com/media/59d0ff_e9812deb74a4408ca77599a19a8f93b6~mv2.jpg)
-![NCC-1701-KB V2 RGB](https://static.wixstatic.com/media/59d0ff_7a8968b70ed849a591664332964cc936~mv2.jpg)
+![NCC1701KBv2](https://imgur.com/a/VoEiNE0)
```

Comment:
```suggestion
![NCC1701KBv2](https://i.imgur.com/Zq1SjBTh.jpg)
```
A link to an Imgur album does not work here — you need a direct link to a single image (with `h` added at the end of the image ID to limit the image size).

It is possible to wrap the image link into a hyperlink to the whole album, but in this case the album contains only a single image, so the album link is not very useful.

Reply:
Ah, ok, I didnt realize that. Thanks for the heads up!